dist/wasm/loader.js

```diff
@@ -61,20 +61,23 @@ function p32(n) {
   return p8((n >>> 24) & 255) + p8((n >>> 16) & 255) + p8((n >>> 8) & 255) + p8(n & 255);
 }
 
-function dump(mem, size) {
```

Comment:
This comes from karamel and https://github.com/FStarLang/karamel/pull/251 right?

Reply:
correct, loader.js gets copied and put under version control in the repository